reference/filter-reference.md

```diff
@@ -82,7 +82,7 @@ spec:
  - `include_body` (optional) controls how much to buffer the request body to pass to the external auth service, for use cases such as computing an HMAC or request signature.  If `include_body` is `null` or unset, then the request body is not buffered at all, and an empty body is passed to the external auth service.  If `include_body` is not `null`, both of its sub-fields are required:
     * `max_bytes` (required) controls the amount of body data that will be passed to the external auth service
     * `allow_partial` (required) controls what happens to requests with bodies larger than `max_bytes`:
-       * if `allow_partial` is `true`, the first `max_bytes` of the body are sent to the external auth service.
+       * if `allow_partial` is `true`, the first `max_bytes` of the body is sent to the external auth service.
```

Comment:
I don't like this change.  "bytes" is plural, and bytes is the subject of the clause.

Reply:
Changed back. But for the record, if there are multiple bytes being sent, they can't all be first (semantically), unless they are together as one unit... making it singular, and therefore making "is" correct

Reply:
![micdrop](https://user-images.githubusercontent.com/55929576/73300428-fbb96d80-41de-11ea-94a9-db70a4ebb149.gif)